src/ploneintranet/workspace/browser/tiles/sidebar.py

```diff
@@ -253,6 +271,25 @@ def children(self):
                 'dpi': dpi})
         return items
 
+    def tasks(self):
+        items = []
+        catalog = getToolByName(self.context, name="portal_catalog")
```

Comment:
If you use the api to get the portal_catalog, as you have done on L166 we can remove the import of getToolByName.